e2etests/features/support/common.rb

```diff
@@ -142,32 +144,32 @@ def tab_click(obj, tab_cnt)
 def check_value(actual, expected)
   if actual != expected
     puts "FAIL Expected: '#{actual}'  Actual: '#{expected}'"
-    @e = "e"
+    @excep = "e"
   end
```

Comment:
When is @excep used ?  If its just the return value of the check_value function then just "e" will be enough, but then, thats always the same, so no point checking it